benchmarks/data_ingestion/benchmark_microgrid_data.py

```diff
@@ -571,6 +571,6 @@ async def benchmark_multiple_formulas(n_formula: int = 100, n_msg: int = 100) ->
 
 
 if __name__ == "__main__":
-    # asyncio.run(benchmark_multiple_meters())
+    asyncio.run(benchmark_multiple_meters())
     # asyncio.run(benchmark_multiple_formulas())
-    asyncio.run(benchmark_multiple_batteries())
+    # asyncio.run(benchmark_multiple_batteries())
```

Comment:
This can probably be left out of the PR, right?

Reply:
Whoops, my bad—I was verifying that the benchmarks are working after making changes to the `MicrogridApiClient` (because they aren't included in the test suite).